rules.txt

```diff
@@ -508,6 +509,7 @@
 /mocs                                     301  /ceo
 /mod                                      301  /departments/mayors-office-for-people-with-disabilities/
 /mois                                     301  /it
+/monkeypox/                               301  /programs/monkeypox/
```

Comment:
here you want to remove the slash at the end of /monkeypox/ in the initial URL so we capture people who navigate without the slash, for example phila.gov/monkeypox